itemframework/src/item/abstract_item.cpp

```diff
@@ -338,27 +372,26 @@ void AbstractItem::clearInputs()
 {
     Q_D(AbstractItem);
 
-    for (int i = d->_inputs.count() - 1; i >= 0; i--) {
```

Comment:
Why was this originally done in reverse? Does doing it in normal order break anything?